Automattic/wp-calypso
# Jetpack Manage: Highlight the 'Licenses' menu item when loading the Issue and Assign License page

Currently, when we load both the Assign and Issue License pages, the sidebar shows the Purchase menu, which doesn't match the Licenses page. This PR fixes this inconsistency.

**Before**
<img width="1947" alt="Screen Shot 2023-11-20 at 4 17 51 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/47de4971-5ddc-4261-a973-fda15b3372a7">
<img width="2183" alt="Screen Shot 2023-11-20 at 4 18 42 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/013634a4-fc14-4db6-a774-dd0844fe50e8">

**After**

<img width="2016" alt="Screen Shot 2023-11-20 at 4 15 22 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/217dc4cf-f6df-43e2-8e76-a7e60a4a99e4">
<img width="2222" alt="Screen Shot 2023-11-20 at 4 16 42 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/b0322c51-a57c-4ec6-a799-3f016675f4b0">


Closes https://github.com/Automattic/jetpack-manage/issues/123

## Proposed Changes

* Update sidebar logic to highlight the **'Licenses'** menu item when loading the Assign or Issue license page.

## Testing Instructions
Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb. Make sure to switch it back to the previous type.

* Use the Jetpack cloud live link below and go to the **Licenses** page (`/partner-portal/issue-license`)
* Click any **'Assign'** button to go to the Assign License page.
* Confirm that the sidebar shows the correct menu items.
* Now go back to the **Licenses** page and press the **'Issue License'** button to go to the Issue License page.
* Confirm that the sidebar shows the correct menu items.

## Pre-merge Checklist
